modal/experimental/flash.py

```diff
@@ -171,19 +173,22 @@ async def close(self):
         if not self.stopped:
             await self.stop()
 
+        await asyncio.sleep(self.exit_grace_period)
+
         logger.warning(f"[Modal Flash] Closing tunnel on {self.tunnel.url}.")
         await self.tunnel_manager.__aexit__(*sys.exc_info())
 
 
-FlashManager = synchronize_api(_FlashManager)
+FlashManager = synchronize_api(_FlashManager, target_module=__name__)
```

Comment:
Does FlashManager need a synchronized interface? Where do we use it from a blocking context?